content/02.introduction.md

```diff
@@ -44,82 +57,70 @@ The current pandemic of COVID-19, caused by the severe acute respiratory syndrom
-
-
-
-[Start with RNA-seq description and summarize studies so far including healthy and infected tissues/cells in vivo/in vitro]
+A second cleavage site within S2 is thought to activate _S_ for fusion by inducing conformational change [@doi:10.1016/j.cell.2020.02.058].
+Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy [@doi:10.1146/annurev-virology-110].
+However, research in SARS-CoV suggests that, in addition to fusion with the plasma membrane, viruses may also be taken up by cells through endocytosis [@doi:10.1038/cr.2008.15].
+<!-- This is very interesting and seems like it could be relevant to virulence (?), but I don't know enough cell biology to expand on it. Another reference that seems useful is doi.org/10.1146/annurev-biochem-060208-104626-->
+Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
+Finally, the dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].
+
+Thus, most of the viral entry process is determined by the viral genome, but the initial step, when the virus binds to a host receptor, also depends on the virus being able to recognize the host receptor.
+Coronaviruses such as SARS-CoV, SARS-CoV-2, and MERS-CoV may therefore be engaged in an evolutionary arms race with the human immune system [@doi:110.1101/2020.05.13.093658; @doi:10.1101/2020.05.14.09613; @doi:10.1186/1743-422X-10-304].
+Two "hot spots" within human ACE2 have been identified as driving the strong binding affinity between this receptor and two sites in SARS-CoV [@doi:10.1146/annurev-virology-110615-042301].
+Coronaviruses bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
+The S1 domain, which contains the RBM, evolves more rapidly than the S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
+However, even within the S1 domain there is variability in conservation.
+The receptors in the N-terminal domain (NTD) evolve more rapidly than those in the C-terminal domain (CTD) [@doi:10.1016/j.jmb.2020.04.009]. 
+Both the NTD and CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
```

Comment:
I think the naming may be a little confusing here. So earlier you say `S1 (N-terminal) subunit and...S2(C-terminal) subunit` so I interpreted that to mean that S1=N-terminal and S2-C-terminal. But now you are talking about S1 N-terminal and S1 C-terminal.